netlify.toml

```
@@ -0,0 +1,14 @@
+[build]
+  command = "yarn workspace client build"
+  functions = "functions"
+  publish = "client/dist"
+
+[build.environment]
+  CONVERSATION_API_HOST = ""
+  LEADERBOARD_API = ""
+  WEBSERVER_URL = ""
+
+[[redirects]]
+    from = "/*"
+    to = "/index.html"
+    status = 200
```

```suggestion
  from = "/*"
  to = "/index.html"
  status = 200
```
Can this have the same indentation as the above?